frontend/src/scenes/insights/InsightHistoryPanel/InsightHistoryPanel.tsx

```diff
@@ -131,71 +133,73 @@ export const InsightHistoryPanel: React.FC<InsightHistoryPanelProps> = ({ displa
     )
 
     return (
-        <div data-attr="insight-history-panel" className="insight-history-panel">
```

Comment:
The only change to this level was to add an indent and wrap with a BindLogic context.

Reply:
Seems like just the right use of `BindLogic` 👌